include/aikido/planner/WorldStateSaver.hpp

```diff
@@ -9,17 +10,29 @@ namespace planner {
 class WorldStateSaver
 {
 public:
-  /// Construct a WorldStateSaver and save the current state of the
-  /// \c World. This state will be restored when
-  /// WorldStateSaver is destructed.
+  /// Options to specify what WorldStateSaver should save.
+  enum Options
+  {
+    CONFIGURATIONS = 1 << 0,
+  };
+
+  /// Construct a WorldStateSaver and save the current state of the \c World.
+  /// This state will be restored when WorldStateSaver is destructed.
   ///
-  /// \param _world World to save state from and restore to.
-  explicit WorldStateSaver(World* const _world);
+  /// \param world World to save state from and restore to.
+  /// \param options Options to specify what should be saved
+  explicit WorldStateSaver(World* const world, int options = CONFIGURATIONS);
```

Comment:
Nit: Please change to `const World* world`. They are identical but different style. I believe our convention is `const World* world`.

Reply:
I think it was actually incorrect to be `const` since we restore the `World` state in the destructor.

Reply:
@brianhou :+1: